CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+* When a transfer fails, the transfer should remain available in the db for at least 12 weeks, as per spec.
```

Comment:
I'd propose to change the description to 84 days (e.g. "at least 84 days, as per spec") to be consistent with the TTL format (P84D). This way it will be less confusing for the users of the adaptor.

Reply:
I see what you mean, but I also want to reflect what's said in the spec. So I'm changing it to: 
"When a transfer fails, the transfer should remain available in the db for at least 12 weeks (84 days), as per spec."
I think this helps to clarify for both spec and devs.